index.html

```diff
@@ -316,6 +316,11 @@ <h3>BrowserCaptureMediaStreamTrack</h3>
                 If [=this=] is not a [=self-capture video track=], the user agent MUST return a new
                 {{Promise}}, [=rejected=] with an {{NotSupportedError}}.
               </li>
+              <li>
+                If the user agent is unable to change the [=crop-state=] of the
+                {{MediaStreamTrack}}, for example due to resource limitation, the user agent MUST
+                return a new {{Promise}}, [=rejected=] with an {{InvalidStateError}}.
```

Comment:
@alvestrand suggested using `OperationError` for this, which I agree with (I'd like to reserve `InvalidStateError` to replace the current `NotAllowedError` which I've suggested elsewhere should be done).
```suggestion
                return a new {{Promise}}, [=rejected=] with an {{OperationError}}.
```
Also, this should be moved to the in-parallel steps, since it is a processing failure, not an input validation failure.